routes/api.js

```diff
@@ -0,0 +1,26 @@
+var express = require('express');
+var router = express.Router();
+
+var bodyParser = require('body-parser');
+
+router.post('/api/user-access', function(req, res) {
+    var stationState = 'Enabled';
+    
+    if(!req.body || req.body.length === 0 || req.body > 200) {
+        stationState = 'Disabled';
+        res.set({
+            'Content-Type': 'text/plain',
+            'Station-State': stationState,
+            'Status': res.status(400)
```

Comment:
I believe this should be moved to a separate call, something like this:
`res.status(400).end()`
I still haven't written a line of node, so I may have the syntax wrong. the main thing though is that response status isn't a header, it's its own special field in the response stream.
https://expressjs.com/en/4x/api.html#res.status